app/common/SessionKeys.scala

```diff
@@ -22,6 +22,7 @@ object SessionKeys {
   val validationEmailKey = "validationEmail"
   val prepopulationWebsiteKey = "vatCorrespondencePrepoulationWebsite"
   val validationWebsiteKey = "validationWebsite"
+  val websiteKey = "Website"
```

Comment:
Do we need this extra session key? AFAIK we only need the two keys above for the website; one for the form validation and one for pre-pop. When you submit the form then it should add the value to the pre-pop key.
While you're here, would you mind updating the `validationWebsiteKey` value to be `vatCorrespondenceValidationWebsite`? We were pulled up on having session keys that weren't specific enough before.